examples/project_configuration/model_chat_evaluation_project.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
<p><code>dataset_name</code>: The name of the dataset <strong>were</strong> the data rows" -&gt; typo: "where"</p><p> </p><p>"Optional description of your project." -&gt; For consistency "<strong>The</strong> optional description of your project"</p><p> </p><p> </p><p>Suggestion: "A dataset ID of an existing Labelbox dataset." -&gt; "An <strong>optional</strong> dataset ID of an existing Labelbox dataset."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1645/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>